src/pages/home/ReportScreen.js

```diff
@@ -227,6 +231,10 @@ class ReportScreen extends React.Component {
         // (which is shown, until all the actual views of the ReportScreen have been rendered)
         const animatePlaceholder = !freeze;
 
+        const defaultViewStyle = [styles.flex1, styles.justifyContentEnd, styles.overflowHidden];
+
+        const mobileViewStyle = [styles.overflowHidden, {height: this.state.viewportHeightWithKeyboard - 52}];
```

Comment:
@shonsirsha Can you please explain number '52' being used in the above line?

We don't prefer direct use of a number in the logic. If it can be replaced by some variables from [variable.js](https://github.com/Expensify/App/blob/main/src/styles/variables.js), let's do it.

Also, please do add some comments to explain why two different styles are needed.

Reply:
ah, sorry I missed putting this info in the proposal. Firstly, this should've been 65 not 52.

65 is the total height of the compose component (including all of its spacings):

![Screenshot 2023-01-23 at 07 20 51](https://user-images.githubusercontent.com/28676406/213977054-d6d858e3-f639-4392-871c-9c3f6d965007.png)

`65px` is a combination of: `variables.inputSizeNormal` and the `marginTop`, `marginBottom`, and `height` of `chatItemComposeSecondaryRow`.

I've also added comments as to why we needed these two styles. Please take another look. Thanks.